mvebu64boot.c

```diff
@@ -751,7 +753,11 @@ static int open_tty(const char *device)
 	int flags;
 	int fd;
 
-	printf("Opening tty device '%s'\n", device);
+	if (strcmp(device, "-") == 0) {
+		return atoi(device+1);
```

Comment:
What a nonsense is this code? strcmp returns zero when device[0] is '-' and device[1] is nul term. So this code is passing always empty string to atoi function, which expects string with number.

Reply:
mvebu64boot sets everything to establish correct connection. Due to precise timings and settings, this must be done in tool itself, to ensure that transfer would work correctly. This is not generic tool for communication with random software on the other side, like lszrz. This is tool special for marvell bootroms. It requires direct tty device, not some random filters.